### ISSUE TITLE
warnings for refresh icon in player

### ISSUE THREAD

**Issue by sure1243jupiter** [abbasfreestyle/react-native-af-video-player#66]
versions :
"react-native": "0.54.1",
"react-native-af-video-player": "^0.2.1",

getting the error since the last two versions of video player
getting this screen and the refresh icon is failing to displaying its color

![vidplayer_issue1](https://user-images.githubusercontent.com/39368870/42306170-9e725098-804b-11e8-8fec-6dfd141dce94.PNG)

The warning says the error is getting in video.js file in the "react-native-af-video-player" library

![vidplayer_issue2](https://user-images.githubusercontent.com/39368870/42306180-acc9a254-804b-11e8-961b-0937f3103777.PNG)

and also getting this warning after that

![vidplayer_issue3](https://user-images.githubusercontent.com/39368870/42306186-b57062c6-804b-11e8-9992-9b6e225b57be.PNG)
![vidplayer_issue4](https://user-images.githubusercontent.com/39368870/42306187-b59bbae8-804b-11e8-83f2-340a5df606b4.PNG)

**Comment by cleoTang** [comment]
Hi! Now I have this problem, can I ask you how to solve it?

**Comment by sure1243jupiter** [comment]
Actually i have not been using RN Since long time but any way try these solutions.

```
const theme = {
    title: '#FFF',
    more: '#FFF',
    center: '#FFF',
    fullscreen: '#FFF',
    volume: '#FFF',
    scrubberThumb: '#FFF',
    scrubberBar: '#FFF',
    seconds: '#FFF',
    duration: '#FFF',
    progress: '#FFF',
    loading: '#FFF'
  }
```

1. try declaring this theme in your file and pass this theme as prop to Video
```
<Video
theme={theme}
/>
```
2. or if it doesn't work try directly modifying the library file video.js file in  "node_modules/react-native-af-video-player/components"
 
add a custom theme like this 
```
const theme = {
    title: '#FFF',
    more: '#FFF',
    center: '#FFF',
    fullscreen: '#FFF',
    volume: '#FFF',
    scrubberThumb: '#FFF',
    scrubberBar: '#FFF',
    seconds: '#FFF',
    duration: '#FFF',
    progress: '#FFF',
    loading: '#FFF',
   iconColor: '#FFF'   // this is the icon color
  }
```
and pass this theme as props in <Video>
and in Video.js file library in line no.318 you will find a tag with <Icons>

change there like this and save the file.
```
  <Icons
          name="replay"
          size={60}
          color={this.props.theme.iconColor}
          onPress={() => this.setState({ renderError: false })}
/>
```


3. or of that doesn't work try directly adding color to <Icons> without depending on props but this will make the icon player color to static we cannot change it dynamicaly then from props.

```
 <Icons
          name="replay"
          size={60}
          color="#FFF"
          onPress={() => this.setState({ renderError: false })}
/>
```

pls comment out if this works or not.

**Comment by cleoTang** [comment]
Thank you, I try to

**Comment by cleoTang** [comment]
The second solution can solve my problem. Thank you.
